arborcat.module

```diff
@@ -1,120 +1,216 @@
 <?php
+
+use Drupal\Core\Datetime\DrupalDateTime;
+
 /**
  * Display help and module information
  * @return help text for section
  */
-function arborcat_help($path, $arg) {
-  $output = '';
+function arborcat_help($path, $arg)
+{
```

Comment:
Coding standards: opening curly brace on the same line as function declaration. Pedantic, I know.
https://www.drupal.org/docs/develop/standards/coding-standards#functdecl